admin-ui/src/lib/admin-token.tsx

```
@@ -4,18 +4,33 @@ const ADMIN_TOKEN_STORAGE_KEY = "adminToken"
+  }
+
+  return ""
 }
 
 export function writeAdminToken(token: string): void {
   if (typeof window === "undefined") return
 
   const trimmed = token.trim()
   if (trimmed) {
-    window.sessionStorage.setItem(ADMIN_TOKEN_STORAGE_KEY, trimmed)
+    window.localStorage.setItem(ADMIN_TOKEN_STORAGE_KEY, trimmed)
```

ESLint

[error] Prefer `globalThis` over `window`.

(unicorn/prefer-global-this)